final_prep/mod_zero_hero.rb

```diff
@@ -2,40 +2,91 @@
 
 # Declare two variables - hero_name AND special_ability - set to strings
 
+hero_name = "Jupiter"
+special_ability = "Telekenesis"
+
 # Declare two variables - greeting AND catchphrase
+
+greeting = "Hello! I am #{hero_name}!"
+catchphrase = "#{special_ability} helps me painlessly remove cats from trees!"
+
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
 
 # Declare two variables - power AND energy - set to integers
 
+power = 20
+energy = 350
+
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
 
+full_power = power * 500
+full_energy = energy + 150
+
 # Declare two variables - is_human and identity_concealed - assigned to booleans
 
+is_human = false
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
+
+arch_nemesis = ["Night Man", "Galaxy Brain", "Super Nova"]
+sidekicks = ["Pluto", "Saturn", "Venus"]
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
 
 # Print the first sidekick to your terminal
 
+puts sidekicks[0]
+
 # Print the last arch_enemy to the terminal
 
+puts arch_nemesis[2]
+
 # Write some code to add a new arch_enemy to the arch_enemies array
 
+arch_nemesis.append("Comet Crash")
+
+puts arch_nemesis
+
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
 
 # Remove the first sidekick from the sidekicks array
 
+sidekicks.delete_at(0)
+sidekicks.append("Barnacle Boy")
+
 # Print the sidekicks array to terminal to ensure you added a new sidekick
 
+print sidekicks
+
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
 
+
+def assess_situation(danger_level)
```

Comment:
I believe the instructions call for the `assess_situation` function to take in **three arguments** (see L34).